docs/mint.json

```diff
@@ -268,6 +273,137 @@
+        "event-catalog/historical.data.vo2_max.created"
+      ]
+    },
+    {
+      "group": "Vitals Timeseries",
+      "pages": [
+        "event-catalog/daily.data.blood_pressure.created",
+        "event-catalog/daily.data.blood_pressure.updated",
+        "event-catalog/daily.data.blood_oxygen.created",
+        "event-catalog/daily.data.blood_oxygen.updated",
+        "event-catalog/daily.data.cholesterol.created",
+        "event-catalog/daily.data.cholesterol.updated",
+        "event-catalog/daily.data.glucose.created",
+        "event-catalog/daily.data.glucose.updated",
+        "event-catalog/daily.data.ige.created",
+        "event-catalog/daily.data.ige.updated",
+        "event-catalog/daily.data.igg.created",
+        "event-catalog/daily.data.igg.updated",
+        "event-catalog/daily.data.heartrate.created",
+        "event-catalog/daily.data.heartrate.updated",
+        "event-catalog/daily.data.hrv.created",
+        "event-catalog/daily.data.hrv.updated",
+        "event-catalog/historical.data.blood_pressure.created",
+        "event-catalog/historical.data.blood_oxygen.created",
+        "event-catalog/historical.data.glucose.created",
+        "event-catalog/historical.data.heartrate.created",
+        "event-catalog/historical.data.hrv.created"
+      ]
+    },
+    {
+      "group": "Body Timeseries",
+      "pages": [
+        "event-catalog/daily.data.fat.created",
+        "event-catalog/daily.data.fat.updated",
+        "event-catalog/daily.data.weight.created",
+        "event-catalog/daily.data.weight.updated",
+        "event-catalog/historical.data.fat.created",
+        "event-catalog/historical.data.weight.created"
+      ]
+    },
+    {
+      "group": "Nutrition Summaries",
+      "pages": [
+        "event-catalog/daily.data.meal.created",
+        "event-catalog/daily.data.meal.updated",
+        "event-catalog/historical.data.meal.created"
+      ]
+    },
+    {
+      "group": "Nutrition Timeseries",
+      "pages": [
+        "event-catalog/daily.data.caffeine.created",
+        "event-catalog/daily.data.caffeine.updated",
+        "event-catalog/daily.data.water.created",
+        "event-catalog/daily.data.water.updated",
+        "event-catalog/historical.data.caffeine.created",
+        "event-catalog/historical.data.water.created"
+      ]
+    },
+    {
+      "group": "Wellness Timeseries",
+      "pages": [
+        "event-catalog/daily.data.hypnogram.created",
+        "event-catalog/daily.data.hypnogram.updated",
+        "event-catalog/daily.data.mindfulness_minutes.created",
+        "event-catalog/daily.data.mindfulness_minutes.updated",
+        "event-catalog/daily.data.respiratory_rate.created",
+        "event-catalog/daily.data.respiratory_rate.updated",
+        "event-catalog/historical.data.hypnogram.created",
+        "event-catalog/historical.data.mindfulness_minutes.created",
+        "event-catalog/historical.data.respiratory_rate.created"
```

Comment:
I would only leave mindfulness in this category. All the others should go to timeseries.